tests/bootstrap.php

```diff
@@ -38,6 +38,11 @@
 include_once __DIR__ . '/../inc/includes.php';
 include_once __DIR__ . '/DbTestCase.php';
 
+// check folder exists instead of class_exists('\GuzzleHttp\Client'), to prevent global includes
+if(!file_exists(__DIR__ . '/../vendor/guzzlehttp/guzzle')){
```

Comment:
@remicollet, see this rule for phpcs `Squiz.ControlStructures.ControlSignature` to automate it